semantic-source/semantic-source.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:       2.4
 
 name:                semantic-source
-version:             0.1.0.0
+version:             0.0.3.0
```

Comment:
This goes back, not forward, in the versioning history. It should probably be 0.2.0.0, yeah? (Might this have been the source of weirdness in your local copy all along?)

Reply:
I don't really understand why this is `0.1.0.0` because the [hackage package](https://hackage.haskell.org/package/semantic-source) lists the last published package version as `0.0.2.0`